charts/logging-operator-logging/templates/eventtailer.yaml

```
@@ -0,0 +1,20 @@
+{{- with $.Values.eventTailer }}
+apiVersion: logging-extensions.banzaicloud.io/v1alpha1
+kind: EventTailer
+metadata:
+  name: {{ .name | default "EventTailer" }}
```

shouldn't the name be DNS subdomain naming compatible?
```suggestion
  name: {{ .name | default "event-tailer" }}
```